Jenkinsfile

```
@@ -97,7 +97,12 @@ pipeline {
                       (source ../zephyr/zephyr-env.sh && \
                       pip install --user -r ../tools/ci-tools/requirements.txt && \
                       pip install --user pylint && \
-		      echo "<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>" > compliance.xml)
+		      echo "<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>" > compliance.xml
```

took a second look at this.
should it actually have been:
```suggestion
echo "<?xml version=\\\"1.0\\\" encoding=\\\"utf-8\\\"?>" > compliance.xml
```
as the first `\\` ensures a `\` goes into the file, and the second `\"` ensures `"` goes into the file.

---

I don't think so, I checked the generated file and `\\"` works fine with Groovy/Jenkins

---

See https://gist.github.com/Faheetah/e11bd0315c34ed32e681616e41279ef4